Sources/LLBuildManifest/BuildManifest.swift

```diff
@@ -96,13 +96,17 @@ public struct BuildManifest {
         inputs: [Node],
         outputs: [Node],
         args: [String],
+        environ: [String: String] = [:],
```

Comment:
nit: environ -> environment (more swifty)

Reply:
Agreed.  I was trying to follow the style of "args" above.  I can change this but would probably also want to change '`args` etc.